src/serial-kernels.lisp

```diff
@@ -170,10 +170,16 @@ Note: This function is raw and unsafe. Pass bad data => get a crash."
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;; One Qubit ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-#-(and qvm-intrinsics avx2)
+#-(and qvm-intrinsics
+       #.(cl:if (sb-alien:extern-alien "avx2_supported" sb-alien:int)
```

Comment:
Kind of gross but reasonable. Have you thought about pushing our own private feature somewhere else to eliminate all these calls?

Reply:
Do you think it's acceptable to push :QVM-AVX2 to cl:*FEATURES* and check that instead?

Reply:
This is the approach I'll take for now.